src/geometry/CTiglRelativelyPositionedComponent.cpp

```diff
@@ -113,7 +113,19 @@ void CTiglRelativelyPositionedComponent::SetSymmetryAxis(const TiglSymmetryAxis&
 CTiglTransformation CTiglRelativelyPositionedComponent::GetTransformationMatrix() const
 {
     const CTiglTransformation thisTransformation = _transformation ? _transformation->getTransformationMatrix() : CTiglTransformation();
-    return thisTransformation;
+    if (_parent) {
+        const auto& parentTransformation = _parent->GetTransformationMatrix();
+
+        // we only transform with the parent's translation
```

Comment:
Add only parent translation, if absLocal!

Reply:
i see. i must have missed this one.